osm/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/osm/WorkflowOSM.groovy

```diff
@@ -588,11 +590,10 @@ IProcess osm_processing() {
                                      results.putAll(geoIndicators.getResults())
                                 }
                             }
-
+                            def x_size = grid_indicators_params.x_size
+                            def y_size = grid_indicators_params.y_size
```

Comment:
Why do you define x_size and y_size before verifying that grid_indicators_params is set ?

Reply:
to avoid null variable in the save method

Reply:
OK, but what happens if grid_indicators_params is not set ?

Reply:
oupps. Good catch

Reply:
I suggest this instead of lines 593 to 596 by
```
                            def x_size
                            def y_size
                            //Compute the grid indicators
                            if(grid_indicators_params){
                                 x_size = grid_indicators_params.x_size
                                 y_size = grid_indicators_params.y_size
```

Then, in saveOutputFiles at line 1098, the condition should be modified as :
```
            if(!xsize ||!ysize || x_size!=y_size){
```
instead of 
```
            if(x_size!=y_size){
```
What do you think of this, @ebocher ?